test/functional/cursor.test.js

```diff
@@ -4117,59 +4119,6 @@ describe('Cursor', function() {
     }
   });
 
-  it('Correctly decorate the collection cursor count command with skip, limit, hint, readConcern', {
```

Comment:
why was this test deleted?

Reply:
Currently taking another look at this test actually, deleting it may have been a mistake.

Reply:
Yeah it was a bug that was causing this test to fail, fixed now.